packages/eas-cli/src/credentials/manager/SelectPlatform.ts

```diff
@@ -20,7 +20,7 @@ export class SelectPlatform {
   ) {}
 
   async runAsync(): Promise<void> {
-    const platform = await selectPlatformAsync(this.flagPlatform);
+    const platform = await selectPlatformAsync(this.flagPlatform, true /* allowExit */);
```

Comment:
Just curious: why are we passing in a parameter here if it's hard coded to `true`? Is there a future-proofing or code consistency reason?